packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.spec.js

```diff
@@ -61,4 +63,17 @@ describe('Component > InteractionLayer', function () {
     wrapper.simulate('pointerdown', fakeEvent)
     expect(mockDrawingTask.activeTool.createMark).to.have.been.calledOnce()
   })
+
+  it('should place a new mark on pointer down', function () {
+    expect(mockMark.initialPosition).to.have.been.calledOnce()
+  })
+
+  it('should drag the new mark on pointer down + move', function () {
+    const fakeEvent = {
+      type: 'pointer'
+    }
+    wrapper.simulate('pointerdown', fakeEvent)
+    wrapper.simulate('pointermove', fakeEvent)
+    expect(mockMark.initialDrag).to.have.been.calledOnce()
+  })
```

Comment:
The drawing API inside the `InteractionLayer` has changed a lot and, I think, needs a bunch of tests. I'm also still inclined to rename it to `DrawingCanvas`.

Reply:
I think the original idea for `InteractionLayer` is that it would be capturing all events of interactivity, not just for drawing, but if this is deviating from that idea, I'd be ok with a name change. We'll have to note that in an update to the ADR.

Reply:
Will the tests be updated in a follow up PR?

Reply:
I _think_ my comment was written before I wrote these new tests. I'm not sure. I guess finishing mark creation (pointer up) is still missing tests?

Reply:
>I think the original idea for InteractionLayer is that it would be capturing all events of interactivity, not just for drawing, but if this is deviating from that idea, I'd be ok with a name change. We'll have to note that in an update to the ADR.

In PFE, each task carries a reference to the React component that should be used to render annotations for that task eg. [`DrawingTask.SVGRenderer`](https://github.com/zooniverse/Panoptes-Front-End/blob/3d842ffabc009857ed86b720382439455c025fb8/app/classifier/tasks/drawing/index.cjsx#L13)

We can copy that idea across to the task models, so that the interaction layer is available outside of the drawing task.

That said, this should be spec'ed somewhere. I've been working to the idea that this component is responding specifically to drawing task events. If that's not correct, we should catch it while it can still be changed.